app/routes/items/new.js

```diff
@@ -69,78 +54,98 @@ export default AuthorizeRoute.extend({
     });
   },
 
-  async setupController(controller, model) {
+  setupController(controller, model) {
     this._super(controller, model);
+    const store = this.get("store");
+    this.initializeController();
+
+    if (this.get("newItemRequest")) {
+      this.initializeDimensions();
+      this.manageSubformDetails();
+      this.setUpPackageImage();
+      window.localStorage.setItem("isSelectLocationPreviousRoute", false);
```

Comment:
I know this is probably old legacy code, but that is sadly not an acceptable use of `localStorage`, nor is it the right way to structure navigation. I suggest removing it and using a default browser back instead.

It might lead to inconsistencies/bugs but we'll fix those when they actually do happen. If we want to go from hell to heaven, we still need to walk through some fire 😄